tests/saul/Makefile

```diff
@@ -5,4 +5,7 @@ USEMODULE += saul_default
 
 USEMODULE += xtimer
 
+# Too little flash:
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-uno
```

Comment:
The flash increase comes from these two commits: https://github.com/RIOT-OS/RIOT/pull/10755/commits/45511d4a8bf250decdb99e6ba1aeb7a0b6c36968 https://github.com/RIOT-OS/RIOT/pull/10755/commits/30b350b19cc2f19bf875a4a0e7f9bba331fff17b

These boards will not have `USEMODULE += ltc4150` in `tests/saul` enabled, so this is not the drivers fault. It's really only the strings of the units and sensors type that increase the ROM.

Reply:
the above was compiled and run with your changes on an Arduino-Uno, here build output:

```
BOARD=arduino-uno make -C tests/saul/
Building application "tests_saul" for "arduino-uno" with MCU "atmega328p".

"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/boards/arduino-uno
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/boards/common/arduino-atmega
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/boards/common/atmega
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/core
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/cpu/atmega328p
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/cpu/atmega_common
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/cpu/atmega_common/periph
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/drivers
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/drivers/periph_common
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/drivers/saul
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/auto_init
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/auto_init/saul
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/div
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/fmt
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/isrpipe
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/phydat
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/saul_reg
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/stdio_uart
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/tsrb
"/Library/Developer/CommandLineTools/usr/bin/make" -C /Volumes/devel/github/smlng/RIOT/sys/xtimer
   text	   data	    bss	    dec	    hex	filename
  10382	    906	    981	  12269	   2fed	/Volumes/devel/github/smlng/RIOT/tests/saul/bin/arduino-uno/tests_saul.elf

```

though is done on macOS with following versions:

```

Operating System Environment
-----------------------------
       Operating System: Mac OS X 10.14.2
                 Kernel: Darwin 18.2.0 x86_64 i386

Installed compiler toolchains
-----------------------------
             native gcc: gcc (Homebrew GCC 8.2.0) 8.2.0
      arm-none-eabi-gcc: arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
                avr-gcc: avr-gcc (GCC) 8.2.0
       mips-mti-elf-gcc: missing
             msp430-gcc: missing
   riscv-none-embed-gcc: missing
   xtensa-esp32-elf-gcc: missing
   xtensa-lx106-elf-gcc: missing
                  clang: Apple LLVM version 10.0.0 (clang-1000.10.44.4)

Installed compiler libs
-----------------------
   arm-none-eabi-newlib: "2.5.0"
    mips-mti-elf-newlib: missing
riscv-none-embed-newlib: missing
xtensa-esp32-elf-newlib: missing
xtensa-lx106-elf-newlib: missing
               avr-libc: "2.0.0" ("20150208")

Installed development tools
---------------------------
                  cmake: cmake version 3.13.2
               cppcheck: Cppcheck 1.86
                doxygen: 1.8.15
                 flake8: 3.6.0 (mccabe: 0.6.1, pycodestyle: 2.4.0, pyflakes: 2.0.0) CPython 3.7.2 on Darwin
                    git: git version 2.20.1
                   make: GNU Make 3.81
                openocd: Open On-Chip Debugger 0.10.0
                 python: Python 2.7.15
                python2: Python 2.7.15
                python3: Python 3.7.2
             coccinelle: spatch version 1.0.6 compiled with OCaml version 4.05.0
```

Could be that it does not fit using some other compiler version on Linux. However, if it is necessary to pass Murdock leave it in - I don't like these BOARDS_XXX list anyway and such should be handled differently by the build system ... but that's another story (or PR).

Reply:
I can confirm that the build is too big for arduino-uno and arduino-duemilanove on my local setup, and alow with BUILD_IN_DOCKER=1

Reply:
I added this because of a Murdock build fail in the previous PR. As the strings of the units and the sensors have not been changed, it should still be necessary.

> I don't like these BOARDS_XXX list anyway and such should be handled differently by the build system 

+1 on that. E.g. if unrelated code gets improved for ROM size previously blacklisted boards will likely remain blacklisted, even if that would no longer be possible. (But I personally do not have an idea how a better solution would look like...)

Reply:
oki, than leave the boards in there